### DISCUSSION TITLE
Recreating this in Replit

### DISCUSSION THREAD

**Discussion by ghost** [jagrosh/MusicBot#1060] [Questions] [answered]
Hello awesome developers of this Music Bot!

Just wondering if i could ask the what the starting command is, because i am actually doing this on replit! 

<3 , RealMaoMao

**Comment by jagrosh** [comment, marked as answer]
As per the page with hosting recommendations (https://jmusicbot.com/hosting/), replit and similar platforms **are not supported**.
